### ISSUE TITLE
chatbox_scrolldiv is undefined

### ISSUE THREAD

**Issue by pmeier** [Quansight/ragna#231]
### Bug description

When I open the developer tools after starting the Ragna UI, I see

```
Uncaught TypeError: chatbox_scrolldiv is undefined
```

### How to reproduce the bug?

Start the Ragna UI and look at the browser console.

### Anything else?

This is coming from

https://github.com/Quansight/ragna/blob/b2730f1dd2b7285a13301cb3e8a7936eca7bd0bb/ragna/deploy/_ui/central_view.py#L521-L545

**Comment by pmeier** [comment]
I think this is no longer relevant after #261, but let's make sure.

**Comment by pmeier** [comment]
#261 removed the offending code and with it the error.
